### DISCUSSION TITLE
What is the only 🇺🇸 state that officially has a royal palace?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#856] [Galaxy Trivia] [answered]
**Bonus:** What is hidden in the palace basement?

**Comment by peterbaccaro** [comment]
The only state in the United States that officially has a royal palace is **Hawaii**. The Iolani Palace, located in Honolulu on the island of Oahu, served as the royal residence for the Hawaiian monarchs. It was commissioned by King David Kalakaua and completed in 1882. The palace was a symbol of Hawaiian monarchy and served as the political and cultural hub of the Kingdom of Hawaii until the overthrow of the monarchy in 1893. Today, Iolani Palace is a historic site and museum, preserving the history and heritage of the Hawaiian monarchy.

The basement has a gallery exhibit of select **hidden treasures from the Royal Collection**, the restored Kitchen and the Chamberlain’s Office.

**Reply by ParanoidUser** [reply]
That's right. And what about the time before it became a museum?

**Reply by peterbaccaro** [reply, marked as answer]
Before it became a museum, ʻIolani Palace served as the official residence of the Hawaiian monarchs and played a significant role in the political and cultural life of the Kingdom of Hawaii.

The basement served various functional purposes related to the daily operations and maintenance of the palace. Some of the uses of the basement included:

- **Kitchen**: The basement area housed the palace kitchen, where meals were prepared for the royal family and their guests. It contained cooking ranges, storage areas for food and supplies, and other kitchen amenities.

- **Servant Quarters**: The basement provided living quarters for the palace staff, including servants, maids, and other employees who worked in the palace. These quarters offered sleeping and living spaces for the staff members.

- **Storage Areas**: The basement served as a storage space for various items, including kitchen supplies, cleaning supplies, linens, and other household goods used in the palace. It provided a convenient location to store and organize these essential items.

- **Laundry Room**: The basement contained a dedicated area for laundry, where the palace's laundry and linens were washed, ironed, and maintained. This space was used to ensure the cleanliness and upkeep of the palace's textiles.
